resources/sparql_queries/q1.rq

```diff
@@ -1,32 +0,0 @@
-PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```

Comment:
Why is this file deleted? Where are the queries now?

Reply:
For now, the resources are in dags/resources/sparql_queries. They will be put back in the main folder, after some changes in DAGs. For this, it is necessary to discuss the TDA architecture.